server.js

```diff
@@ -70,7 +73,7 @@ passport.use('provider', new OAuth2Strategy(
     authorizationURL: oauth.authorizationUrl,
     tokenURL: oauth.tokenUrl,
     clientID: oauth.clientId,
-    clientSecret: oauth.clientSecret,
+    clientSecret: process.env.CLIENT_SECRET,
```

Comment:
I am absolutely, totally, 110% down with moving the secrets to the `.env` file though — thank you very much for doing that. 

But I'm not quite seeing why it's better to assign directly from `process.env.CLIENT_SECRET` here, rather than in the `appConfig.js`

Not a big deal.

Reply:
(You changed it back for this one in a later PR — I don't think it really matters which approach you take, but I think it's best to be consistent)

Reply:
You know what? I'm sorry — I changed my mind. I think it does make sense to refer to it via `process.env.CLIENT_SECRET`, as it's really clear that you shouldn't add it to the config file. I think it's just important to be consistent.